### ISSUE TITLE
I deleted facebook......can I still create bots without a facebook account?

### ISSUE THREAD

**Issue by danuluma** [devc-nairobi/ntw-hello-bot#6]
I deleted facebook......can I still create bots without a facebook account?

**Comment by Benedict-Mambo** [comment]
No. You need to have a facebook account and a page for that matter. So, go ahead and create a facebook account to create your bot.
